### PR TITLE
Feature/va 013 post request

### PR DESCRIPTION
**Pull request by agonzalesb107**

### CODE REVIEW

**Comment by agonzalesb107** [comment]
[VA-013 [SP-3 ] : Create Post endpoint that sends text requests](https://trello.com/c/J7Jp1Ogm/27-va-013-sp-3-create-post-endpoint-that-sends-text-requests)

File: sql/V1_03__create_request_table.sql
Comment (zamoranoalicia) [inline sql/V1_03__create_request_table.sql:1]: All the sql files are already in the repository, why you are rewriting the files again

File: src/main/java/org/fundacionjala/virtualassistant/models/RequestEntity.java [hunk 1]
```diff
@@ -1,58 +1,42 @@
...
+import javax.persistence.Table;
+import java.time.ZonedDateTime;
 import java.util.Date;
 
 @Entity
+@Data
+@NoArgsConstructor
+@Builder
 @Table(name = "request")
+@AllArgsConstructor
```
Comment (zamoranoalicia) [inline src/main/java/org/fundacionjala/virtualassistant/models/RequestEntity.java:22]: Mmove this above to the @Table annotation

File: src/main/java/org/fundacionjala/virtualassistant/textrequest/service/TextRequestService.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    RequestEntityRepository requestEntityRepository;
+
+    public TextRequestResponse createTextRequest(TextRequest textRequest) throws TextRequestException {
+        if (textRequest.getIdUser() == null || textRequest.getIdUser() <= 0) {
+            throw new TextRequestException(TEXT_REQUEST_USER_ID_NULL);
+        }
+
+        RequestEntity requestEntity = requestEntityFromTextRequest(textRequest);
+        RequestEntity savedRequestEntity = requestEntityRepository.save(requestEntity);
+
```
Comment (zamoranoalicia) [inline src/main/java/org/fundacionjala/virtualassistant/textrequest/service/TextRequestService.java:26]: remove this unnecessary new lines

File: src/test/java/org/fundacionjala/virtualassistant/repository/RequestEntityRepositoryTest.java [hunk 3]
```diff
@@ -4,6 +4,7 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import java.time.ZonedDateTime;
 import java.util.Date;
 
 import static org.junit.jupiter.api.Assertions.*;
```
Comment (zamoranoalicia) [inline src/test/java/org/fundacionjala/virtualassistant/repository/RequestEntityRepositoryTest.java:10]: Dont use wildcards, please use specific class names

File: src/test/java/org/fundacionjala/virtualassistant/textrequest/service/TextRequestServiceTest.java
Comment (zamoranoalicia) [inline src/test/java/org/fundacionjala/virtualassistant/textrequest/service/TextRequestServiceTest.java:1]: Please add more unit tests, and you have to compare the results from the  service, also add unit tests for the controller

**Review by zamoranoalicia — CHANGES_REQUESTED** [review]

**Pull request merged**